### PR TITLE
Add r5py to conda-forge

### PR DESCRIPTION
**Pull request by HTenkanen**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by HTenkanen** [comment]
> * License is not an SPDX identifier (or a custom LicenseRef) nor an SPDX license expression.
> 
> Documentation on acceptable licenses can be found [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#spdx-identifiers-and-expressions).

@conda-forge/help-python: Not sure how the license information / files should be added to get rid of this warning/suggestion?

**Comment by HTenkanen** [comment]
@conda-forge/staged-recipes : The recipe is ready to go!

File: recipes/r5py/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  license_file:
+    - LICENSE
+    - licenses/
+  doc_url: https://r5py.readthedocs.io/
+  dev_url: https://github.com/r5py/r5py
+
+extra:
+  recipe-maintainers:
+    - htenkanen
+    - christophfink
```
Comment (ocefpaf) [inline recipes/r5py/meta.yaml:54]: @christophfink can you confirm you are OK being a mantainer here?

File: recipes/r5py/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  license: GPLv3 or later, MIT
+  license_file:
+    - LICENSE
+    - licenses/
+  doc_url: https://r5py.readthedocs.io/
+  dev_url: https://github.com/r5py/r5py
+
+extra:
+  recipe-maintainers:
+    - htenkanen
```
Comment (ocefpaf) [inline recipes/r5py/meta.yaml:53]: @HTenkanen can you use the correct capitalization of your GitHub handle?

File: recipes/r5py/meta.yaml (hunk 1 above)
Comment (christophfink) [reply recipes/r5py/meta.yaml:54]: Yes, I confirm that I'm okay with being a maintainer.

File: recipes/r5py/meta.yaml (hunk 2 above)
Comment (HTenkanen) [reply recipes/r5py/meta.yaml]: @ocefpaf Well spotted! Done ✅

**Comment by HTenkanen** [comment]
@conda-forge/staged-recipes : The recipe is ready for review.

**Pull request merged**